### ISSUE TITLE
Is there a way to customize a custom toolbar?

### ISSUE THREAD

**Issue by Przemek625** [ckeditor/strapi-plugin-ckeditor#33]
Is there a way to create a customize toolbar?

How can I enable `forcePasteAsPlainText`?

https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-forcePasteAsPlainText

**Comment by MokDevelopment** [comment]
Yeah until we cannot prevent users from selecting images, italic or h1 tags, layouts can get killed and we need editors who check gainst seo rules. 

It is really bad practice to allow users without seo knowledge to define h tags - as they use it mostly for design purposes - which is also bad as now there is no corporate uniform design style for all articles. 

Instead it would be better to allow classes... like

h2.main
h3.sub

remove h1 as the site title is already h1
Also tables should be an advanced feature. 

Can we not hook somehow in between to use the fantastic features of ckeditor?

**Comment by H-Gomez** [comment]
Anyone one the ckeditor team looking into this?
